app/views/schools/add_participants/name.html.erb

```diff
@@ -0,0 +1,22 @@
+<% content_for :title, "Enter full name to confirm" %>
+
+<% content_for :before_content, govuk_back_link(text: "Back", href: back_link_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl"><%= @school.name %> - <%= @cohort.display_name %> cohort</span>
+
+    <% if add_participant_form.type == :ect %>
+      <h1 class="govuk-heading-xl">What’s the full name of this ECT?</h1>
```

Comment:
This would probably be better as an option to the text field, i.e.

```ruby
legend: { text: "text goes here", size: "xl", tag: "h1" }
```

Generates more semantically correct HTML, and better layout

Reply:
to include the caption, use a proc: https://govuk-form-builder.netlify.app/introduction/labels-captions-hints-and-legends/#input-erb-fully-customising-legends-with-procs

Reply:
Given we aren't doing this anywhere else in the codebase, I would suggest we create a separate ticket for this one and update them all for consistency

Reply:
Follow up tickets created https://dfedigital.atlassian.net/browse/CPDRP-1022 && https://dfedigital.atlassian.net/browse/CPDRP-1023